edi_sale_ubl_oca/tests/test_order_response_out.py

```diff
@@ -76,4 +82,4 @@ def test_xml(self):
         self.assertEqual(err, None, err)
         data = handler.parse_xml(file_content)
         # TODO: test all main data
-        self.assertEqual(data["cbc:OrderResponseCode"], "AP")
+        self.assertEqual(data["cbc:OrderResponseCode"], "CA")
```

Comment:
Can put a comment something like `Order has not been created from exc record in (just linked to record out), so all lines are not in Accepted state`

You can see [here](https://github.com/OCA/edi-framework/blob/669c46b9d44291c59aa85bc8a95ca83f72e97c5f/edi_record_metadata_oca/models/edi_exchange_consumer_mixin.py#L23) --> no store metadata to change the state in lines